MAUI/PDF-Viewer/Printing-PDF-files.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Printing PDF files in .NET MAUI PDF Viewer control | Syncfusion
+description: Learn here all about printing feature in Syncfusion .NET MAUI PDF Viewer (SfPdfViewer) control and more.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+---
+
+# Printing PDF Files in .NET MAUI PDF Viewer (SfPdfViewer)
+
+![Printing PDF Files in .NET MAUI PDF Viewer](Images\Print\print.png)
```

Comment:
Screenshots need to be added at the end of the paragraph

Reply:
Added